hw1/02-palindrome.js

```diff
@@ -1,3 +1,30 @@
-const elem = document.querySelector('input');
+const inputElement = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+inputElement.addEventListener("input", isNumPalindrome);
+
+function isNumPalindrome() {
+  var num = document.getElementById("number").value;
```

Comment:
Hello, you should consider changing the use of var to let or const depending on if you're updating num or not.